Workbooks/CyberArkEPV.json

```diff
@@ -0,0 +1,155 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## !!IMPORTANT NOTES!!\n\nDue to the current way data is being presented the CEF strandard Custom Label functionality is not displaying properly. Here are the Custom Labels and their new column labels\n\n|      Old Label     |       Sentinel Label      |    xsl KeyName   |\n|:------------------:|:-------------------------:|:----------------:|\n| Safe Name          | FileName                  | fname            |\n| Device Type        | FileType                  | fileType         |\n| Affected User Name | SourceUserPrivileges      | spriv            |\n| Database           | DeviceExternalID          | deviceExternalId |\n| Other info         | DestinationUserPrivileges | dpriv            |\n| Request Id         | FileID                    | fileId           |\n| Ticket Id          | OldFileID                 | oldFileId        |\n"
+      },
+      "name": "CyberArk-Workbook-Notes"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\n| where OldFileID contains \"Error\"| summarize AggregatedValue = count() by bin(TimeGenerated, 1h)| sort by TimeGenerated desc| render timechart",
```

Comment:
This is the example of a modified query per my feedback for modifying all queries in this workbook. Please prefix all queries to have "\n| where DeviceVendor == "Cyber-Ark" \n| where DeviceProduct == "Vault" \n|" after the CommonSecurityLog construct.

"CommonSecurityLog\n| \n| where DeviceVendor == \"Cyber-Ark\" \n| where DeviceProduct == \"Vault\" \n| where OldFileID contains \"Error\"| summarize AggregatedValue = count() by bin(TimeGenerated, 1h)| sort by TimeGenerated desc| render timechart"